specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json

```diff
@@ -53,7 +53,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the gallery."
+            "description": "The name of the Shared Image Gallery. The allowed characters are alphabets and numbers with dots, dashes, and periods allowed in the middle. The maximum length is 80 characters as limited by Constants.MaxResourceNameLength for any type of resource created in Azure."
```

Comment:
Don't need this "as limited by Constants.MaxResourceNameLength for any type of resource created in Azure." as people don't know what Constants.MaxResourceNameLength is